src/main/resources/data/minecolonies/colony/quests/tutorial/builder2.json

```diff
@@ -98,7 +98,7 @@
       "unlocks-rewards": [0],
       "type": "minecolonies:dialogue",
       "target": 0,
-      "text": "Awesome! This will now also allow you to upgrade all kinds of other buildings. Make sure to regularly check the quest log about future tasks!",
+      "text": "Awesome! This will now also allow you to upgrade all kinds of other buildings. If you think progress is too slow, consider getting a second builder. Make sure to regularly check the quest log about future tasks!",
```

Comment:
Make sure to regularly check the quest log about future tasks!
Think it would sound better as "Make sure to regularly check the quest log for future tasks!"